oca_dependencies.txt

```diff
@@ -3,3 +3,4 @@ connector
 account-invoicing
 account-financial-tools
 l10n-spain
+contract
```

Comment:
Can you split the change to oca_dependencies in a different commit?

Reply:
Done